build.assets/build-arch.sh

```diff
@@ -0,0 +1,150 @@
+#!/bin/bash
+
+set -e
+
+export GOLANG_VERSION=go1.21.5
+
+export RUST_VERSION=1.71.1
+
+# Switch depending on the architecture which is passed as the script argument
+case $1 in
+    "arm64")
+        export ARCH="arm64"
+        export GO_ARCH="arm64"
+        export SYSROOT="${HOME}/x-tools/aarch64-centos7-linux-gnu/aarch64-centos7-linux-gnu/sysroot"
```

Comment:
Is `$HOME` the appropriate directory for this?

Reply:
We could move it to `/opt` but for hack MVP $HOME is good enough 😅